monitoring/tracer/run_poll_locally.sh

```diff
@@ -16,12 +18,13 @@ echo Reminder: must be run from root repo folder
 docker build \
     -f monitoring/tracer/Dockerfile \
     -t interuss/dss/tracer \
-    --build-arg version=`scripts/git/commit.sh` \
+    --build-arg version="$(scripts/git/commit.sh)" \
     monitoring
 
+# shellcheck disable=SC2086
```

Comment:
Don't the quotes below fix this issue?

Reply:
Same as above; If we quote the variable as proposed by the linter, $TRACER_OPTIONS is passed as a single argument (escaping word splitting) and the python script fails to properly parse arguments.